sanic/server/protocols/base_protocol.py

```diff
@@ -90,8 +90,9 @@ def close(self, timeout: Optional[float] = None):
         if self.transport:
             self.transport.close()
             if timeout is None:
-                timeout = self.app.config.GRACEFUL_SHUTDOWN_TIMEOUT
-            self.loop.call_later(timeout, self.abort)
+                self.abort()
```

Comment:
This may have inadvertent consequences by immediately closing the connection instead of waiting for the default timeout.

Reply:
absolutely agree. however, I was unable to simulate such a situation, but it was easy to simulate a situation when there were a bunch of objects left that the garbage collector did nothing with

Reply:
ok, maybe this really isn't a good change

Reply:
any updates? I've removed this diff